ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -11966,7 +11966,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build03
-  cron: 12 14 27 * *
```

Comment:
I saw there is no such change in job config, why here it is.

Reply:
This is changed automatically when I run `make update`

Reply:
got it.